src/widgets/Header/index.tsx

```diff
@@ -1,22 +1,74 @@
 'use client';
-import s from './style.module.scss';
-import Logo from '@/assets/images/logo.svg';
+import { clsx } from 'clsx';
+import Link from 'next/link';
+import { useState, useCallback } from 'react';
+
+import { CancelIcon } from '@/assets/icons/CancelIcon';
 import LogoBlue from '@/assets/images/logo-blue.svg';
-import LogoSmall from '@/assets/images/logo-small.svg';
-import { Container } from '@/components/Container';
+import LogoMobile from '@/assets/images/logo-mobile.svg';
+import Logo from '@/assets/images/logo.svg';
+import Hamburger from '@/assets/images/hamburger.svg';
+import HamburgerBlue from '@/assets/images/hamburger-blue.svg';
+
 import { Button } from '@/components/Button';
-import { HamburgerIcon } from '@/assets/icons/HamburgerIcon';
-import { clsx } from 'clsx';
+import { Container } from '@/components/Container';
 import { NavLink } from '@/components/NavLink';
-import Link from 'next/link';
+
+import s from './style.module.scss';
+import SocialMedia from '@/widgets/SocialMedia';
 
 interface HeaderProps {
   whiteBg?: boolean;
   isHome?: boolean;
 }
 
+type NavItem = {
+  href: string;
+  label: string;
+};
+
+const navItems: NavItem[] = [
```

Comment:
We should also show a link on the mobile nav for `Contact Us` that goes to that page.